events/mbed_shared_queues.cpp

```diff
@@ -15,7 +15,10 @@
  */
 
 #include "events/mbed_shared_queues.h"
-#include "mbed.h"
+
+#ifdef MBED_CONF_RTOS_PRESENT
```

Comment:
I can see similar changes in this PR that could be separated from adding new config (changes in events files at least)? As I expect with this config change to touch only mbed or std namespace (add it ,remove it).

Reply:
Removing namespace i.e building mbed-os with "-DMBED_NO_GLOBAL_USING_DIRECTIVE" results in build failures as 'mbed.h' added here will not add RTOS / Mbed namespace in cpp files as well. 

This cleanup "i.e. add only required namespace/header file" is needed for removal of namespace to work. I can add it as separate PR, and make this PR to be dependent on that.